libsolidity/analysis/DeclarationTypeChecker.cpp

```diff
@@ -62,6 +62,18 @@ bool DeclarationTypeChecker::visit(ElementaryTypeName const& _typeName)
 	return true;
 }
 
+bool DeclarationTypeChecker::visit(EnumDefinition const& _enum)
+{
+	if (_enum.members().size() > 256)
+		m_errorReporter.fatalTypeError(
+			1611_error,
+			_enum.location(),
+			"enum with more than 256 members is not allowed."
```

Comment:
```suggestion
			"Enum with more than 256 members is not allowed."
```

Reply:
Right. I did this for consistency with the error for zero members (https://github.com/ethereum/solidity/blob/develop/libsolidity/parsing/Parser.cpp#L675) but I probably should have just capitalized both.

Reply:
Done.